pages/flop.js

```diff
@@ -89,14 +97,24 @@ const Index = () => {
                 }}
               >
                 <Text variant="h2">Active Auctions</Text>
-                <Button
-                  variant="pill"
-                  sx={{ ml: 5 }}
-                  disabled={!web3Connected}
-                  onClick={() => fetchAuctions(maker)}
-                >
-                  Sync
-                </Button>
+                {
+                  isSyncing 
+                    ? <Spinner
+                        sx={{
+                          height: '32px',
```

Comment:
no hardcoded values, ever, use the theme.

Reply:
fixed @adrianleb